src/services/helper/sessions.js

```diff
@@ -292,6 +295,7 @@ module.exports = class SessionsHelper {
 				message: message,
 			})
 		} catch (error) {
+			console.log(error)
```

Comment:
@nevil-mathew use logger instead of console.log

Reply:
removed